worlds/witness/rules.py

```diff
@@ -196,6 +196,8 @@ def _has_item(item: str, world: "WitnessWorld",
     if item == "Entity Hunt":
         # Right now, panel hunt is the only type of entity hunt. This may need to be changed later
         return _can_do_panel_hunt(world)
+    if "Eggs" in item:
```

Comment:
```suggestion
    if "Egg" in item:
```
To go along with the previous suggestion